obs_api.py

```diff
@@ -0,0 +1,130 @@
+
+MSG_SEPARATOR = '|'
+MSG_FINAL_SEPARATOR = '                                     '
+
+class ObsFooter:
+
+	def __init__(self, field_id, field_name=''):
+
+		self.field_id = field_id
+
+		if field_name != '':
+			self.field_name = field_id
+
+		self.field_name = field_name
+		self.file_name = field_id + '.txt'
+		self.content_list = []
+
+		write_to_file(self.file_name)
+
+	def add_message(self, author_id, message):
+		self.content_list.append([author_id, message])
+		self.parse_to_file()
+
+		return True
+
+	def remove_message(self, msg_id):
+		
+		if msg_id > len(self.content_list):
+			return False
+		
+		else:
+			del self.content_list[msg_id]
+			self.parse_to_file()
+
+			return True
+
+	def get_message_ids(self):
+		if len(self.content_list) < 1:
+			return ('Não há mensagens inseridas no rodapé\n')
+		
+		msg = 'Lista de mensagens inseridas no rodapé: \n'
+		msg += '<autor> - <id> - <mensagem>\n'
+		
+		for content in self.content_list:
+			msg += ('<@' + str(content[0]) + '>' + ' - ' + str(self.content_list.index(content)+1) + ' - "' + content[1] + '"\n')
+
+		return msg
+
+	def parse_to_file(self):
+		footer = ''
+
+		if len(self.content_list) > 0:
+			for i in range(len(self.content_list)-1):
+				footer += self.content_list[i][1] + ' ' + MSG_SEPARATOR + ' '
+		
+		footer += self.content_list[len(self.content_list)-1][1] + MSG_FINAL_SEPARATOR
+
+		write_to_file(self.file_name, footer)
+
+
+class ObsField:
+
+	def __init__(self, field_id, field_name=''):
+
+		self.field_id = field_id
+
+		if field_name != '':
+			self.field_name = field_id
+
+		self.field_name = field_name
+		self.file_name = field_id + '.txt'
+		self.content = []
+
+		write_to_file(self.file_name)
+
+	def add_message(self, author_id, message):
+		self.content = [author_id, message]
+		self.parse_to_file()
+
+		return True
+
+	def remove_message(self, msg_id=0):
+		
+		if len(self.content) < 2:
+			return False
+		
+		else:
```

Comment:
Get rid of the `else:`, you're returning.

Reply:
Fixed